### PR TITLE
Feat/icalendar file support

### PR DESCRIPTION
**Pull request by mgerhold**

### CODE REVIEW

File: app/Controllers/Event.php [hunk 1]
```diff
@@ -75,4 +75,51 @@ public function get(int|null $year = null) {
             'talks' => $talks,
         ]);
     }
+
+    public function getICalendarFile(int $year) {
+        // Start output buffering
+        ob_start();
```
Comment (peddermaster2) [inline app/Controllers/Event.php:81]: Do you need the output buffer at all? It seems like you are collecting the "output" into the `$icsContent` variable.

Comment (mgerhold) [reply app/Controllers/Event.php:81]: Honestly: I don't know. I took @freddylinux' code and just left this in. If you think that it doesn't make a difference, I can also remove it.

Comment (peddermaster2) [reply app/Controllers/Event.php:81]: From the [documentation](https://www.php.net/manual/en/function.ob-start.php) I read that it is capturing the "script output" aka "echo". And you do not use any of these in that function.

So I would remove it and the `ob_end_clean()` as well.

Comment (PurpurTentakel97) [reply app/Controllers/Event.php:81]: I have no opinion on that.

**Review by PurpurTentakel97 — APPROVED** [review]

File: app/Controllers/Event.php (hunk 1 above)
Comment (mgerhold) [reply app/Controllers/Event.php:81]: > From the [documentation](https://www.php.net/manual/en/function.ob-start.php) I read that it is capturing the "script output" aka "echo". And you do not use any of these in that function.
> 
> So I would remove it and the `ob_end_clean()` as well.

Sounds reasonable, will remove it 👍

**Review by PurpurTentakel97 — APPROVED** [review]

**Pull request merged**